encrypted-config-value-bundle-dropwizard1/src/main/java/com/palantir/config/crypto/EncryptConfigValueCommand.java

```diff
@@ -60,6 +61,6 @@ public void run(Bootstrap<?> bootstrap, Namespace namespace) throws Exception {
         EncryptedValue encryptedValue = algorithm.getEncryptedValue(value, keyWithAlgorithm);
 
         // print the resulting encrypted value to the console
-        System.out.println(encryptedValue);
+        System.out.println(encryptedValue.serialize());
```

Comment:
Checkstyle's design-for-extension lint and Immutables's setup become kind of unhappy with overriding toString at the `EncryptedValue` level, but I think I can hammer on things enough to make it work.